testsuite/MDAnalysisTests/topology/test_itp.py

```diff
@@ -219,6 +291,11 @@ def test_mass_parse_or_guess(self, universe):
         assert_allclose(universe.atoms[2].mass, 100.0)
         assert_allclose(universe.atoms[3].mass, 100.0)
 
+    @pytest.mark.skip(reason="TPParser doesn't guess types")
+    def test_guessed_types(self, filename, guessed_types):
+        pass
```

Comment:
This is a good reason to rewrite the `test_guessed_types` function to use `top` and check `self.guessed_attrs` before testing -- could avoid this construction!